src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs

```diff
@@ -38,24 +39,99 @@ public class Organisation :
         IElasticEventHandler<PurposeUpdated>,
         IElasticEventHandler<OrganisationTerminationSyncedWithKbo>,
         IElasticEventHandler<OrganisationTerminated>,
-        IElasticEventHandler<OrganisationTerminatedV2>
+        IElasticEventHandler<OrganisationTerminatedV2>,
+        IElasticEventHandler<OrganisationPlacedUnderVlimpersManagement>,
+        IElasticEventHandler<OrganisationReleasedFromVlimpersManagement>
     {
+        private readonly IOrganisationManagementConfiguration _organisationManagementConfiguration;
         private readonly Elastic _elastic;
         private readonly ElasticSearchConfiguration _elasticSearchOptions;
 
         public Organisation(
             ILogger<Organisation> logger,
             Elastic elastic,
-            IOptions<ElasticSearchConfiguration> elasticSearchOptions) : base(logger)
+            IOptions<ElasticSearchConfiguration> elasticSearchOptions,
+            IOrganisationManagementConfiguration organisationManagementConfiguration) : base(logger)
         {
             _elastic = elastic;
             _elasticSearchOptions = elasticSearchOptions.Value;
+            _organisationManagementConfiguration = organisationManagementConfiguration;
         }
 
-        public async Task<IElasticChange> Handle(DbConnection dbConnection, DbTransaction dbTransaction, IEnvelope<OrganisationCreated> message)
+        public async Task<IElasticChange> Handle(
```

Comment:
hmmm is this really domain logic ? somehow feels out of place in the AR. This is code I would place in the application layer I think.